onvif/examples/camera.rs

```diff
@@ -207,11 +208,9 @@ async fn get_system_date_and_time(clients: &Clients) {
     println!("{:#?}", date);
 }
 
-async fn get_stream_uris(clients: &Clients) {
+async fn get_stream_uris(clients: &Clients) -> Result<(), transport::Error> {
     let media_client = clients.media.as_ref().unwrap();
```

Comment:
If we are returning a result, as good practice, we should return all the errors, not just the transport one and let the other unwrap.